package.json

```diff
@@ -39,5 +39,14 @@
 		"erc20-tokens",
 		"erc20",
 		"erc721"
-	]
+	],
+	"scripts": {
+		"manifest": "node scripts/manifest.js",
+		"precommit": "npm run manifest"
```

Comment:
There's no built-in `precommit` hook. I think you want https://github.com/typicode/husky

Reply:
`precommit`, i'm using the hook of git.

Reply:
There's no `precommit` hook built-in: https://docs.npmjs.com/misc/scripts